cmd/config.go

```diff
@@ -263,6 +259,12 @@ func cfgFilesAdd(dir string) (cfg *Config, err error) {
 		}
 		fmt.Printf("added chain %s...\n", c.ChainID)
 	}
+	for _, p := range cfg.Paths {
+		if err = config.ValidatePath(p); err != nil {
+			fmt.Println("Error occured...Config is not updated")
```

Comment:
```suggestion
```
maybe we could wrap the error and say the path validation failed

Reply:
Yeah Sure, but we are displaying added chain logs before. That's the reason, I wrote `Config is not updated`